src/libmongoc/src/mongoc/mongoc-server-description.c

```diff
@@ -822,6 +826,8 @@ mongoc_server_description_new_copy (
    memcpy (&copy->error, &description->error, sizeof copy->error);
 
    copy->generation = description->generation;
+   copy->generation_map =
+      mongoc_generation_map_copy (description->generation_map);
    return copy;
```

Comment:
If mongoc_generation_map_copy() is updated to return NULL on failure, remember to consider whatever needs to happen to invalidate "copy" or notify of error.